test/core/event_engine/cf/cf_engine_test.cc

```diff
@@ -80,6 +82,211 @@ TEST(CFEventEngineTest, TestConnectionCancelled) {
   client_signal.WaitForNotification();
 }
 
+namespace {
+std::vector<std::string> ResolvedAddressesToStrings(
+    const std::vector<EventEngine::ResolvedAddress> addresses) {
+  std::vector<std::string> ip_strings;
+  std::transform(addresses.cbegin(), addresses.cend(),
+                 std::back_inserter(ip_strings), [](auto const& address) {
+                   return ResolvedAddressToString(address).value_or("ERROR");
+                 });
+  return ip_strings;
+}
+}  // namespace
+
+TEST(CFEventEngineTest, CFEventEngineTest_TestCreateDNSResolver_Test) {
```

Comment:
`CFEventEngineTest_` in the name is redundant, it can just be `TEST(CFEventEngineTest, CreateDNSResolver)`. I believe this will show up in bazel test logs as `CFEventEngineTest.CreateDNSResolver`.